app/Http/Kernel.php

```diff
@@ -37,12 +38,14 @@ class Kernel extends HttpKernel
             \Illuminate\Session\Middleware\StartSession::class,
             \Illuminate\View\Middleware\ShareErrorsFromSession::class,
             \App\Http\Middleware\VerifyCsrfToken::class,
+            TeamsPermission::class,
```

Comment:
We need the middleware in the "web" group, *or* manually added to the same group of routes as the `org.selected` middleware. We don't need both.  (We definitely don't need it in the API middleware, as we don't use any API routes)

I would probably keep it linked to the group of routes that org.selected runs on.

Reply:
I see. I will remove it from "api" group.